manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -6012,6 +6012,11 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "watch", "list"]
+
+  # Used for MCS serviceexport management
+  - apiGroups: ["multicluster.x-k8s.io"]
+    resources: ["serviceexports"]
+    verbs: ["create", "delete"]
```

Comment:
are you sure we only want these? and not get/watch/list as well?

Reply:
IIRC that was a comment on the original PR as well. I have no problem with that ... probably makes sense.

Reply:
I can add some context there - this is the minimal set of permissions for the ServiceExport functionality to work. Eventually, we'll need get/watch/list to support ServiceImport, but I wanted to add the smallest set of access I could. I see no problem expanding it prematurely though, as we know ServiceImport will happen soon